Server.js

```diff
@@ -415,6 +415,14 @@ Server.prototype = {
 			capabilities.browserVersion = capabilities.version;
 		}
 
+		//WebDriver Spec now supports execute/sync as the synchronous execute endpoint.
+		// This was updated in geckodriver as of 16.0 but not yet completed in chromedriver.
+		capabilities.syncEndpoint = false;
```

Comment:
Maybe `executeSyncEndpoint` or `useExecuteSyncEndpoint`

Reply:
Fixed to the latter.